examples/src/main/scala/org/apache/spark/examples/mllib/MovieLensALS.scala

```diff
@@ -39,14 +39,15 @@ import org.apache.spark.rdd.RDD
 object MovieLensALS {
 
   case class Params(
-      input: String = null,
-      kryo: Boolean = false,
-      numIterations: Int = 20,
-      lambda: Double = 1.0,
-      rank: Int = 10,
-      numUserBlocks: Int = -1,
-      numProductBlocks: Int = -1,
-      implicitPrefs: Boolean = false) extends AbstractParams[Params]
+    input: String = null,
```

Comment:
I think it went back to 2-space indentation. The old code has correct indentation. Please do not change.